config/config/sc-config.yaml

```diff
@@ -1040,6 +1040,27 @@ welcomingDashboard:
 
 # Network policies for service cluster
 networkPolicies:
+
+  global:
+    apiserver:
+      ips: # usualy private ip of control-plane nodes
+        - set-me
+      port: 6443
+    scNodes:
+      ips: # ip of all nodes in the cluster for internal communication
+        - set-me
+    wcIngress:
+      ips: # ip(s) to wc loadbalancer if available, otherwise wc worker nodes
+        - set-me  
+    scIngress:
+      ips: # ip(s) to sc loadbalancer if available, otherwise sc worker nodes
+        - set-me    
```

Comment:
This should be moved to common-config since we want the same in both clusters

Reply:
moved!!